app/javascript/controllers/keywords_controller.js

```diff
@@ -18,36 +18,17 @@ export default class extends Controller {
     this.containerTargets.forEach((container) => container.classList.remove('keywords-container-open') )
   }
 
-  add(event) {
-    const content = this.selectedTemplateTarget.innerHTML
-      .replace(/TEMPLATE_RECORD/g, new Date().valueOf())
-      .replace(/TEMPLATE_LABEL/g, event.target.value)
-
-    this.searchTarget.value = ''
-    this.addItemTarget.insertAdjacentHTML('beforeend', content)
-    // Remove error indications when keyword(s) added
-    this.containerTarget.classList.remove('is-invalid')
-    this.checkForDuplicates()
-  }
-
   checkForDuplicates() {
```

Comment:
this methods isn't called anywhere from any components in use as far as I can tell, instead we will just call `checkForDuplicates` directly from the existing component in use